src/app/modules/login/pages/login-page/login-page.component.ts

```diff
@@ -22,14 +25,30 @@ export class LoginPageComponent implements OnInit {
     this.authService
       .login(user.email, user.password)
       .then((result) => {
-        console.log(result);
         this.loading$.next(false);
-        this.authService.setUserStorage(result.user);
-        this.router.navigate(['/todo/list']);
+        this.handleLoginSuccess(result.user);
+      })
+      .catch(() => {
+        this.errorForm$.next(true);
+        this.loading$.next(false);
+      });
+  }
+
+  public loginGoogle(): void{
+    this.authService
+      .loginGoogle()
+      .then((result) => {
+        this.loadingGoogle$.next(false);
+        this.handleLoginSuccess(result.user);
       })
       .catch(() => {
         this.errorForm$.next(true);
         this.loading$.next(false);
       });
   }
+
+  private handleLoginSuccess(user: User): void{
+    this.authService.setUserStorage(user);
```

Comment:
save an user information is not responsible of this component. Make that in authService